tools/perf/Makefile

```diff
@@ -0,0 +1,131 @@
+TOP := $(shell pwd)
+TOOLS_DIR = $(HOME)/bin
+CURL_PATH ?= curl
+CURL_DOWNLOAD := $(CURL_PATH) --location --fail --progress-bar
+
+KIND_CLUSTER_NAME ?= kuma
+KIND_KUBECONFIG_DIR := $(HOME)/.kube
+KIND_KUBECONFIG := $(KIND_KUBECONFIG_DIR)/kind-$(KIND_CLUSTER_NAME)-config
+HELM_PATH := $(TOOLS_DIR)/helm
+KIND_PATH := $(TOOLS_DIR)/kind
+KUBECTL_PATH := $(TOOLS_DIR)/kubectl
+HELM_VERSION ?= v3.8.2
+KIND_VERSION ?= v0.14.0
+KUBECTL_VERSION ?= v1.23.5
+KUMA_VERSION ?= 1.8.0
+
+# wrk test configuration
+WRK_CONN ?= 100
+WRK_THREAD ?= 12
+WRK_DURATION ?= 60
+export PATH := $(TOOLS_DIR):$(PATH)
+CONFIG_PATH := single-node
+ifdef USE_MULTI_NODE
```

Comment:
nitpick: this is the only place where `USE_MULTI_NODE` is used (if my search is correct) why not use 

```
CONFIG_PATH ?= single-node
```

then?

Reply:
I am using `USE_MULTI_NODE ` to set `CONFIG_PATH ` and based on this I am using different config files to create kind cluster `--config "$(TOP)/$(CONFIG_PATH)/kind-config.yaml" \`